src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -453,6 +453,37 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->registerForAutoconfiguration(RouteLoaderInterface::class)
             ->addTag('routing.route_loader');
+
+        $this->addClassesToPreload([
+            'Psr\Log\LogLevel',
```

Comment:
Why not use `::class` here? I see it used for `registerForAutoconfiguration` calls.

Reply:
`use` statements should be used for things that the code actually uses.
Having the list as ordinary strings will also help to maintain it, by reducing merge conflicts related to `use` statements management across branches.

Reply:
`use` statements should be used for things that the code actually uses.
Having the list as ordinary strings will also help to maintain it, by reducing merge conflicts related to `use` statements management across branches.

Reply:
Didn't think of that, thanks for the explanation 😃

Reply:
Can you elaborate on the maintainability when using ::class? If it really is about use statements, why not using the fully qualified class here, but still the ::class statement. 
The benefit I‘d see is that searching for usages or refactoring of classes would be much more straightforward (even if some IDEs might be able to find references of a class in strings)

Reply:
Good catch @helhum. Do you have any comments on that @nicolas-grekas?

```diff
- 'Psr\Log\LogLevel'
+ \Psr\Log\LogLevel::class
```